spec/timezone.html

```diff
@@ -269,8 +290,7 @@ <h1>Temporal.TimeZone.prototype.toJSON ( )</h1>
       </p>
       <emu-alg>
         1. Let _timeZone_ be the *this* value.
-        1. Let _toString_ be ? Get(_timeZone_, *"toString"*).
-        1. Return ? Call(_toString_, _timeZone_).
+        1. Return ? TimeZoneToString(_timeZone_).
```

Comment:
Tests:

- [ ] `toString` is `undefined`
- [ ] `ToString` is called on the result

Reply:
Will add.